hack/install-deps.sh

```diff
@@ -19,32 +22,32 @@ install() {
   fi
 
   if which sha256sum; then
-	echo "found sha256sum"
+	  echo "found sha256sum"
   else
-	echo "Missing sha256sum binary"
-	exit 1
+    echo "Missing sha256sum binary"
+    exit 1
   fi
 
-  ytt_version=v0.38.0
-  kbld_version=v0.32.0
-  kapp_version=v0.45.0
-  imgpkg_version=v0.24.0
-  vendir_version=v0.24.0
+  ytt_version=$(awk '/ytt_version/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+  kbld_version=$(awk '/kbld_version/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+  kapp_version=$(awk '/kapp_version/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+  imgpkg_version=$(awk '/imgpkg_version/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+  vendir_version=$(awk '/vendir_version/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
 
   if [[ `uname` == Darwin ]]; then
     binary_type=darwin-amd64
-    ytt_checksum=1f84e26457a5efd2470df408072bf2f182c939414e4958f82e19e122d2ee120c
-    kbld_checksum=5fc8a491327294717611974c6ab3da2bda3f3809ef3147c1e8472ac62af3ee18
-    kapp_checksum=44f66b7ed664e4537b2caa0453aa239f16e20c028b11fedc22f46f3cac3ba9fd
-    imgpkg_checksum=f0c87c8caefb3d2a82e648779b36783403fe5c93930df2d5cbf4968713933392
-    vendir_checksum=f3a738d1fe55803ad5faba495f662c48efa230976ccad7a159587dcf9b020f63
+    ytt_checksum=$(awk '/ytt_checksum_darwin/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+    kbld_checksum=$(awk '/kbld_checksum_darwin/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+    kapp_checksum=$(awk '/kapp_checksum_darwin/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+    imgpkg_checksum=$(awk '/imgpkg_checksum_darwin/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+    vendir_checksum=$(awk '/vendir_checksum_darwin/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
   else
     binary_type=linux-amd64
-    ytt_checksum=2ca800c561464e0b252e5ee5cacff6aa53831e65e2fb9a09cf388d764013c40d
-    kbld_checksum=de546ac46599e981c20ad74cd2deedf2b0f52458885d00b46b759eddb917351a
-    kapp_checksum=9d2c0441729bebe49e5fb9a1a7b7088f0bee4092ba23e267339a9478948c5731
-    imgpkg_checksum=cfcfcb5afc5e3d28ce1f2f67971a4dcd18f514dadf8a63d70c864e49c9ddca7e
-    vendir_checksum=b7bfd227aa2e6df602f8e79edf725bb0a944b68d207005f42f46f061c4ecd55a
+    ytt_checksum=$(awk '/ytt_checksum_linux/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+    kbld_checksum=$(awk '/kbld_checksum_linux/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+    kapp_checksum=$(awk '/kapp_checksum_linux/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+    imgpkg_checksum=$(awk '/imgpkg_checksum_linux/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
+    vendir_checksum=$(awk '/vendir_checksum_linux/{print $(NF-0)}' "${DEPENDENCIES_DIR}")
```

Comment:
cool cool --  i check and awk is available on the stock katacoda image, but unsurprisingly doesn't seem like it's in photon.  Before merging can you verify real quick that the hack/deploy scripts still work?